src/h5web/visualizations/line/MappedLineVis.tsx

```diff
@@ -37,9 +38,16 @@ function MappedLineVis(props: Props): ReactElement {
     scaleType
   );
 
+  const abscissas =
+    mapperState && axesValues && axesValues[mapperState.indexOf('x')];
```

Comment:
This is the line from which everything stems. Because the only way to determine the dataset to use is its position in the array `axesValues`, I have to keep the number of elements (and its ordering) from the parsing of the `axes` attribute to here.

This means that, at several places, I have to keep `undefined` values in the arrays as will be pointed out in other comments.

Reply:
I feel like this should be moved to `LineVisContainer`. You should simply propagate an `abscissas` prop`.

Reply:
There's something I don't understand, though: how do you know that the index of `x` in `mapperState` matches a dataset (and not a dot) in the `axes` attribute? Doesn't `mapperState` need to be initialized based on `axes`?

Reply:
`axesValues` cannot be `undefined` since you use `slice()`, which always returns an array.